futures-util/src/stream/mod.rs

```diff
@@ -30,6 +32,9 @@ pub use self::collect::Collect;
 mod concat;
 pub use self::concat::Concat;
 
+mod copy_into;
+pub use self::copy_into::CopyInto;
```

Comment:
curious, since this is an IO util type does this need to be put behind a `io` feature flag like `AsyncWrite` is?